README.md

```diff
@@ -19,26 +19,24 @@
 
 ---
 
-## Development
+## Build docs locally
 
-1. Checkout this repository
-1. Install mintlify with `npm i -g mintlify@latest`
-1. Run `mintlify dev`
+1. Fork this repository.
+1. Install mintlify with `npm i -g mintlify@latest`.
+1. Run `mintlify dev`.
 
 ## Deploying 
 
-To build the final product, we have a couple of additional steps:
+To build the final product:
 
-1. Build the JS Protocol Docs
+1. Build the JS Protocol Docs:
 
 - `npm install -g jsdoc-to-markdown`
 - `./bin/jsdocs.sh`
 
-2. Build the PDCP API reference documentation
+2. Build the PDCP API reference documentation:
 
 - Either download the latest `openapi.yaml` manually or run `./bin/download-api.sh`
 - Run `./bin/generate-api.sh` to generate any new API files
 
-3. Deployment
-
-After those, Mintlify handles the deployment automatically.
\ No newline at end of file
+3. Mintlify handles the deployment automatically.
```

Comment:
As a user, this does not tell me how I know if the deployment is successful, or what to do if I don't see updated files.